backend/loader.py

```diff
@@ -30,6 +30,24 @@
 dir_path = os.path.dirname(__file__)
 
 
+def check_huggingface_component(component_name:str, cls_name:str, state_dict):
```

Comment:
in near future, I may be engaged in restoring sd_checkpoints_limit (but not yet a fact). then you will have to work with these modules. so I decided to take a look at the PR.

Spaces? Typehint?
```suggestion
def _check_huggingface_component(component_name: str, cls_name: str, state_dict: dict[str, Any] | list[?]):
```